packages/inference/src/providers/novita.ts

```diff
@@ -50,38 +51,88 @@ export class NovitaConversationalTask extends BaseConversationalTask {
 		return "/v3/openai/chat/completions";
 	}
 }
+
 export class NovitaTextToVideoTask extends TaskProviderHelper implements TextToVideoTaskHelper {
 	constructor() {
 		super("novita", NOVITA_API_BASE_URL);
 	}
 
-	makeRoute(params: UrlParams): string {
-		return `/v3/hf/${params.model}`;
+	override makeRoute(params: UrlParams): string {
+		if (params.authMethod !== "provider-key") {
+			return `/v3/async/${params.model}?_subdomain=queue`;
+		}
+		return `/v3/async/${params.model}`;
```

Comment:
Novita API doesn't have a `queue` subdomain from what I can tell from here: https://novita.ai/docs/guides/model-apis-dedicated-endpoints
```suggestion
		return `/v3/async/${params.model}`;
```